### PR TITLE
fix balance overflow in rpc response

### LINKED ISSUE #218: balance overflow int64
__System info:__ ethermint main branch

__Steps to reproduce:__

1. Set balance of genesis account to `10000000000000000000000aphoton`
2. start devnet and query balance
```
>>> w3.eth.getBalance('0x32A079939Faf05b09798d517Ae4603632c71d432')
1864712049423024128
>>> 10000000000000000000000 % (2**64)
1864712049423024128
```

Probably due to https://github.com/tharsis/ethermint/pull/208/files#diff-0ceec127cab538366d521200eff10d764791b12d19af704a6b4b01e876a90210R39 changed `String` to int64?

__Expected behavior:__ don't overflow

__Actual behavior:__ overflow

### PR DESCRIPTION
**Pull request by yihuang**
Closes #218

## Description

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: ethereum/rpc/namespaces/eth/api.go [hunk 1]
```diff
@@ -215,7 +215,12 @@ func (e *PublicAPI) GetBalance(address common.Address, blockNum rpctypes.BlockNu
 		return nil, err
 	}
 
-	return (*hexutil.Big)(big.NewInt(res.Balance)), nil
+	val, err := ethermint.UnmarshalBigInt(res.Balance)
```
Comment (fedekunze) [inline ethereum/rpc/namespaces/eth/api.go:218]: let's use `sdk.NewIntFromString(res.Balance`

File: ethereum/rpc/namespaces/eth/api.go [hunk 2]
```diff
@@ -992,10 +997,15 @@ func (e *PublicAPI) GetProof(address common.Address, storageKeys []string, block
 		accProofStr = proof.String()
 	}
 
+	balance, err := ethermint.UnmarshalBigInt(res.Balance)
```
Comment (fedekunze) [inline ethereum/rpc/namespaces/eth/api.go:1000]: ditto

File: ethereum/rpc/namespaces/eth/api.go (hunk 1 above)
Comment (yihuang) [reply ethereum/rpc/namespaces/eth/api.go]: fixed, thanks

File: ethereum/rpc/namespaces/eth/api.go (hunk 2 above)
Comment (yihuang) [reply ethereum/rpc/namespaces/eth/api.go]: fixed, thanks.

**Review by fedekunze — APPROVED** [review]

**Pull request merged**